pkgs/tools/X11/devour/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, libX11 }:
+stdenv.mkDerivation rec {
+  version = "12";
+  name = "devour-${version}";
+  src = fetchFromGitHub {
+    owner = "salman-abedin";
+    repo = "devour";
+    rev = version;
+    sha256 = "1qq5l6d0fn8azg7sj7a4m2jsmhlpswl5793clcxs1p34vy4wb2lp";
+  };
+
+  installPhase = ''
+    mkdir -p $out/local/usr/bin
+    mv devour $out/local/usr/bin
```

Comment:
```suggestion
    install -Dm755 devour $out/usr/bin
```

Reply:
That will give give you a devour executable named "bin" and we don't stick stuff in `usr`:

This is the right way:
`install -Dm555 -t $out/bin devour`

Reply:
I will change output to `$out/bin` and permissions to`555`, but I don't think that install command behaves like that. By default it copies compiled sources into destination, which is directory created by the command above (tested). Although I will change it anyway, as most of the other nix packages use `install` with `-t`.

Reply:
Yep, that's much better!